advanced_samples/Dataview_Wind/Jupyter/README.md

```diff
@@ -0,0 +1,74 @@
+# OCS Dataview Data Analysis Sample using Jupyter
+
+**Version:** 1.0.0
+
+[![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/PI-System/PIWebAPI_Data_Analysis?branchName=master)](https://dev.azure.com/osieng/engineering/_build?definitionId=1644&branchName=master)
+
+The sample code in this folder demonstrates how to utilize the OCS Dataviews to do some basic data analysis using Python Jupyter Notebook. In order to run this sample, you need to have [Python](https://www.python.org/downloads/) installed.  
+
+## About the Sample
+
+This sample is intended to show you how you can use an OCS Dataview to bring your data from OCS into Jupyter and easily into Pandas to do machine learning.  Using an OCS Dataview allows you to configure the data you want, in the way you want it, and not have to do as much client side processing of the data.  This is a basic sample that shows you a possible way to do machine learning and a possible outcome of looking at the data.  This is not a guide to Pandas or machine learning.
```

Comment:
Instead of `Dataview` I would generally say `Data View` since I think they're pretty consistently referred to that way.